code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -74,6 +74,8 @@
 	new /obj/item/door_remote/chief_medical_officer(src)
 	new /obj/item/clothing/neck/petcollar(src)
 	new /obj/item/pet_carrier(src)
+	for(var/i in 1 to 3)
+		new /obj/item/wallframe/defib_mount(src)

```

Comment:
I'm going to go out on a limb here and say only put one of these here so they have a reason to actually be made at the lathe